src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.css

```diff
@@ -7,33 +7,21 @@
   min-width: 300px;
 }
 
-.breakpoint-panel-wrapper > :first-child {
-  border-top-left-radius: 4px;
-  border-top-right-radius: 4px;
-}
-
-.breakpoint-panel-wrapper > :last-child {
-  border-bottom-left-radius: 4px;
-  border-bottom-right-radius: 4px;
-}
-
```

Comment:
@jonbell-lot23 not sure if you know if there was a reason we did this and not overflow hidden on the container, but it fixes the background spilling out of the container edges. The only reason I'd see a problem is if we had a popover menu nested inside, but we should be able to just render that at the end of the document to avoid any issues there.